app/main.py

```diff
@@ -2,0 +1,44 @@
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
```

Comment:
The `print_name` method of the `Animal` class could be a `@staticmethod` since it does not use any instance variables. You would just need to pass the name as a parameter.